### ISSUE TITLE
configure's header checking fails.

### ISSUE THREAD

**Issue by olavgun** [mvo5/synaptic#42]
As we see, I do have apt-pkg headers installed, but configure's checking halts because the filee in question "cannot be compiled". 
checking apt-pkg/configuration.h usability... no
checking apt-pkg/configuration.h presence... yes
configure: WARNING: apt-pkg/configuration.h: present but cannot be compiled
configure: WARNING: apt-pkg/configuration.h:     check for missing prerequisite headers?
configure: WARNING: apt-pkg/configuration.h: see the Autoconf documentation
configure: WARNING: apt-pkg/configuration.h:     section "Present But Cannot Be Compiled"
configure: WARNING: apt-pkg/configuration.h: proceeding with the compiler's result
checking for apt-pkg/configuration.h... no
configure: error: You need the apt-pkg headers installed to compile synaptic.

**Comment by trebmuh** [comment]
Did you install the `libapt-pkg-dev` package?

**Comment by olavgun** [comment]
Yes. apt-pkg/configuration.h is present. And that is because is installed from the libapt-pkg-dev package.
The folder /usr/include/apt-pkg is there, full of header-files.

**Comment by trebmuh** [comment]
How are you trying to compile it?
